tests/sentry/api/endpoints/test_organization_invite_request_index.py

```diff
@@ -120,6 +122,18 @@ def test_simple(self):
 
         self.assert_org_member_mapping(org_member=member)
 
+    def test_inviter_must_have_verified_email(self):
+        with assume_test_silo_mode(SiloMode.CONTROL):
+            UserEmail.objects.filter(user=self.user).update(is_verified=False)
+
+        self.login_as(user=self.user)
+
+        response = self.client.post(
+            self.url, {"email": "eric@localhost", "role": "member", "teams": [self.team.slug]}
+        )
+
+        assert response.status_code == 401
```

Comment:
Should we also check the response if it's an actual `email_verification_required` error? Since checking for 401 is a really abstract

Reply:
Ah, yeah, I can check the error details. Good idea. :) 

The frontend doesn't behave as expected, even though we return a 401. No error is actually rendered, so once I have the frontend changes in place, I'll come back to this PR.